data/mimeType.go

```diff
@@ -0,0 +1,461 @@
+package data
+
+// CODE GENERATED AUTOMATICALLY WITH gopkg.in/src-d/enry.v1/internal/code-generator
+// THIS FILE SHOULD NOT BE EDITED BY HAND
+// Extracted from github/linguist commit: d5c8db3fb91963c4b2762ca2ea2ff7cfac109f68
+
+var LanguagesMime = map[string]string{
+	"1C Enterprise":    "",
```

Comment:
Those languages that don't have a mime type shouldn't appear in the map. Skip this entries when generate the file